README.rst

```diff
@@ -103,6 +103,39 @@ Doing a Modbus request requires even less code:
 
     sock.close()
 
+
+The same request can also be made using any asyncio_ compatible StreamReader
+and StreamWriter objects::
+
+    #!/usr/bin/env python
+    # scripts/examples/simple_async_tcp_client.py
+    import asyncio
+
+    from umodbus import conf
+    from umodbus.client import tcp
+
+    # Enable values to be signed (default is False).
+    conf.SIGNED_VALUES = True
+
+
+    async def main():
+        reader, writer = await asyncio.open_connection('localhost', 15020)
```

Comment:
Nitpick: would be nice to use the same 502 port as in the example above

Reply:
Makes total sense. Nice catch!